pharus/component_interface.py

```diff
@@ -290,9 +304,15 @@ def dj_query_route(self):
         record_header, table_records, total_count = _DJConnector._fetch_records(
             query=fetch_metadata["query"] & self.restriction,
             fetch_args=fetch_metadata["fetch_args"],
+            page=int(request.args["page"]) if "page" in request.args else 1,
+            limit=int(request.args["limit"]) if "limit" in request.args else 1000,
```

Comment:
How did this come up? Would be good to understand the context. Perhaps a good default to limit the number of plots might be `10`.

Reply:
that is the default for the _fetch_records method which is why I chose it, but we can set to to 10 if you think that is better.